### DISCUSSION TITLE
Use NWS gif as radar

### DISCUSSION THREAD

**Discussion by robwolff3** [poblabs/weewx-belchertown#817] [Q&A] [answered]
I have a url for my local NWS radar in low resolution gif (https://radar.weather.gov/ridge/lite/KDTX_loop.gif) is it possible to use it in the radar iframe?

When I attempt to I get this:

![](https://i.imgur.com/pj1kMqr.png)

**Comment by norecords** [comment, marked as answer]
Hi try this template has mentioned in issue #703
and adapt to your need.

**Comment by robwolff3** [comment]
That worked beautifully. Thank you!
